### DISCUSSION TITLE
Grains meanOrientation

### DISCUSSION THREAD

**Discussion by KorayKuzkaya** [mtex-toolbox/mtex#1685] [Ask Anything] [answered]
Dear mtex community,

I would like to calculate the trace of a slip system along a grain. In my script, I select a grain for this. mtex uses the mean orientation thats stored for the grainId. Unfortunately, as can be seen in the pictures, the averaging of the orientation of the grain is very rough (the grain should remain separated along the red line). Is there a threshold I can use to set how the averaging proceeds?

![T11area1](https://user-images.githubusercontent.com/50950035/206245918-3180ae99-ded0-42d5-8740-4fa8aeae1f6f.png)
![4239](https://user-images.githubusercontent.com/50950035/206245943-a8a339b1-783f-484d-b74a-2ab206d34f51.png)

**Comment by kilir** [comment, marked as answer]
Hi,
if I understand you correctly, you'd want to use `calcGrains` with a lower '`threshold'` .
Hope that helps.
Rüdiger
